apps/io-session-manager-internal/src/repositories/redis.ts

```diff
@@ -180,5 +208,251 @@ const userHasActiveSessionsOrLV: RTE.ReaderTaskEither<
     ),
   );
 
+const readSessionInfoKeys: RTE.ReaderTaskEither<
+  FastRedisClientDependency & { fiscalCode: FiscalCode },
+  Error,
+  ReadonlyArray<string>
+> = (deps) =>
+  pipe(
+    TE.tryCatch(
+      () =>
+        deps.fastClient.sMembers(
+          `${userSessionsSetKeyPrefix}${deps.fiscalCode}`,
+        ),
+      E.toError,
+    ),
+    TE.chain(
+      TE.fromPredicate(
+        (res): res is NonEmptyArray<string> =>
+          Array.isArray(res) && res.length > 0,
+        () => sessionNotFoundError,
+      ),
+    ),
+  );
+
+const loadSessionBySessionToken: RTE.ReaderTaskEither<
+  FastRedisClientDependency & { token: SessionToken },
+  Error,
+  User
+> = (deps) =>
+  pipe(
+    TE.tryCatch(
+      () => deps.fastClient.get(`${sessionKeyPrefix}${deps.token}`),
+      E.toError,
+    ),
+    TE.chain(
+      flow(
+        O.fromNullable,
+        E.fromOption(() => sessionNotFoundError),
+        E.chain(
+          flow(
+            J.parse,
+            E.mapLeft(E.toError),
+            E.chain(
+              flow(
+                User.decode,
+                E.mapLeft((err) => new Error(readableReportSimplified(err))),
+              ),
+            ),
+          ),
+        ),
+        TE.fromEither,
+      ),
+    ),
+  );
+
+const getUserTokens = (
+  user: User,
+): Record<string, { readonly prefix: string; readonly value: string }> => ({
+  session_info: {
+    prefix: sessionInfoKeyPrefix,
+    value: user.session_token,
+  },
+  session_token: {
+    prefix: sessionKeyPrefix,
+    value: user.session_token,
+  },
+  wallet_token: {
+    prefix: walletKeyPrefix,
+    value: user.wallet_token,
+  },
+  bpd_token: {
+    prefix: bpdTokenPrefix,
+    value: user.bpd_token,
+  },
+  fims_token: {
+    prefix: fimsTokenPrefix,
+    value: user.fims_token,
+  },
+  myportal_token: {
+    prefix: myPortalTokenPrefix,
+    value: user.myportal_token,
+  },
+  zendesk_token: {
+    prefix: zendeskTokenPrefix,
+    value: user.zendesk_token,
+  },
+});
+
+const singleStringReplyAsync = (command: TE.TaskEither<Error, string | null>) =>
```

Comment:
it seems it's not uses, can we remove it?

Reply:
thank you. done in https://github.com/pagopa/io-auth-n-identity-domain/pull/295/commits/a95a334985cfda392c83b9b9d1019748edb19ec4